NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -87,4 +87,13 @@ public extension NeuroID {
             
         NeuroID.saveEventToLocalDataStore(nidEvent)
     }
+    
+    internal static func captureAdvancedDevice() {
```

Comment:
see above suggestion, maybe put the parameter default to NeuroID.isAdvancedDevice